stable/storageos-operator/README.md

```diff
@@ -97,29 +120,15 @@ spec:
 ```
 
 Once the `StorageOSCluster` configuration is applied, the StorageOSCluster
-operator will create a StorageOS cluster in the `storageos` namespace by
+operator will create a StorageOS cluster in the `kube-system` namespace by
 default.
 
-Most installations will want to use the default [CSI](https://kubernetes-csi.github.io/docs/)
-driver.  To use the [Native Driver](https://kubernetes.io/docs/concepts/storage/volumes/#storageos)
-instead, disable CSI:
-
-```yaml
-spec:
-  ...
-  csi:
-    enable: false
-  ...
-```
-
-in the above `StorageOSCluster` resource config.
-
 Learn more about advanced configuration options
 [here](https://github.com/storageos/cluster-operator/blob/master/README.md#storageoscluster-resource-configuration).
 
 To check cluster status, run:
 
-```bash
+```console
 $ kubectl get storageoscluster
```

Comment:
Do we have to add the `-n storageos-operator` to be able to list the CR? Before we used to install the CR in the default NS, I believe that is not the case now.

Reply:
Yes, that's right. It should be that namespace because the `helm install` is also passing a namespace in above.